snmp/datadog_checks/snmp/data/profiles/_generic-router-ip.yaml

```diff
@@ -1,150 +1,4 @@
 # Generic IP metrics for routers
 
```

Comment:
It might help to indicate that those profile with router word are deprecated and kept for backward compatibility.

Reply:
done